.cascade-code/left-nav-refactor/_components/left_nav/left_nav_index_format

```diff
@@ -0,0 +1,164 @@
+## Format used by index pages with the brute-force block that loads the entire file tree.
+
+#if ( $_XPathTool.selectSingleNode( $contentRoot, "//system-folder[@current = 'true']/system-page[@current = 'true']") )
+    #set( $current = true )
```

Comment:
A little confused by `$current` here. In what case might `$current` be false?

I guess one case is the one I was dealing with yesterday when I was working with a page in the `test-section` folder that was marked hidden.

Still, it's a little bit of a struggle to make sense of this here. Anything that can make it clearer (code comment, different var name) will be appreciated by future me.

Reply:
From looking at it and the output of $currentPage I'm inclined to say it would not be false ever. This format is used in a page, which means there will always be a `system-page` in the given block that has the `current='true'` attribute. Maybe @mandythomas has some insight that says otherwise?